src/api/api.ts

```diff
@@ -416,6 +417,41 @@ export class OpenSeaAPI {
     return response;
   }
 
+  /**
+   * Fetch NFTs owned by an account.
+   * @param address The address of the account
+   * @param limit The number of NFTs to retrieve. Must be greater than 0 and less than 51.
+   * @param next Cursor to retrieve the next page of NFTs
+   * @param retries Number of times to retry if the service is unavailable for any reason.
+   * @returns The {@link ListNFTsResponse} returned by the API.
+   */
+  public async getNFTsByAccount(
+    address: string,
+    limit: number | undefined = undefined,
+    next: string | undefined = undefined,
+    retries = 1,
+    { chain = this.chain }: { chain?: Chain } = { chain: this.chain },
```

Comment:
Missing this param in the documentation.
Does this need to be a type? why not `chain: Chain | undefined`?

Reply:
good call, fixed (doing `chain = this.chain` makes the type `chain?: Chain` which accepts undefined btw)